src/scenes/home/getInvolved/getInvolved.css

```diff
@@ -0,0 +1,39 @@
+form {
```

Comment:
Is there a reason for these styles targeting form elements? I don't see any forms in the markup, and also don't see `.pageHeading` used within this component. May not be needed.

Reply:
We also cannot allow element level styles. If there is/are elements you need to apply styles to, inline them or make a class for them.